plugins/module_utils/network/vyos/argspec/bgp_global/bgp_global.py

```diff
@@ -174,7 +174,7 @@ def __init__(self, **kwargs):
                         },
                         "disable_connected_check": {"type": "bool"},
                         "address": {"type": "str"},
-                        "password": {"type": "str"},
+                        "password": {"type": "str", "no_log": True},
```

Comment:
We should have a discussion regarding this. Setting `no_log: True` on keys, would break a significant feature of resource modules - round trip.